middle_end/flambda2/from_lambda/closure_conversion.mli

```diff
@@ -87,7 +87,11 @@ type 'a close_program_metadata =
       * Exported_offsets.t)
       -> [`Classic] close_program_metadata
 
-type 'a close_program_result = Flambda_unit.t * 'a close_program_metadata
+type 'a close_program_result =
+  { unit : Flambda_unit.t;
+    metadata : 'a close_program_metadata;
+    code_slot_offsets : Slot_offsets.t Code_id.Map.t
+  }
```

Comment:
Could we instead add the slot offsets to `Flambda_unit.t` or to the code bindings or something? Then we could naturally add fexpr syntax for them and support them in flexpect.

Reply:
The reason why I didn't do that was so that the slot offsets would **not** end up in cmx files (because it would simply waste space), and unless I'm wrong, adding them to code bindings or flambda units would make them end up in cmx files.